SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -0,0 +1,373 @@
+                    // the load balancing options in the launch config. It creates a rectangular
+                    // launch config of N cols X 1 row, N being the number of coordinators
+                    // to create.
+                    // This assumes the launch config contains a rectangular load balancing
+                    // layer configuration already for the coordinator worker.
+                    var lbLayerConfigurations = simWorkerConfig.load_balancing.layer_configurations;
+                    for (var i = 0; i < lbLayerConfigurations.Count; ++i)
+                    {
+                        if (lbLayerConfigurations[i].layer == CoordinatorWorkerName)
+                        {
+                            var rectangleGrid = lbLayerConfigurations[i].rectangle_grid;
+                            rectangleGrid.cols = simNumPlayers;
+                            rectangleGrid.rows = 1;
+                        }
+                    }
+
+                    simWorkerConfigJson = simWorkerConfig.ToString();
+
+                    // Create simulated player deployment.
+                    var simDeploymentConfig = new Deployment
+                    {
+                        AssemblyId = assemblyName,
+                        LaunchConfig = new LaunchConfig
+                        {
+                            ConfigJson = simWorkerConfigJson
+                        },
+                        Name = simDeploymentName,
+                        ProjectName = projectName
+                        // No snapshot included for the simulated player deployment
+                    };
+
+                    simDeploymentConfig.Tag.Add(SIM_PLAYER_DEPLOYMENT_TAG);
+
+                    Console.WriteLine(
+                        $"Creating the simulated player deployment {simDeploymentName} in project {projectName} with {simNumPlayers} simulated players. Link: https://console.improbable.io/projects/{projectName}/deployments/{simDeploymentName}/overview");
+
+                    var simDeploymentCreateOp = deploymentServiceClient.CreateDeployment(new CreateDeploymentRequest
+                    {
+                        Deployment = simDeploymentConfig
+                    }).PollUntilCompleted();
+
+                    Console.WriteLine("Successfully created the simulated player deployment.");
+                }
+            }
+            catch (Grpc.Core.RpcException e)
+            {
+                if (e.Status.StatusCode == Grpc.Core.StatusCode.NotFound)
+                {
+                    Console.WriteLine(
+                        $"Unable to launch the deployment(s). This is likely because the project '{projectName}' or assembly '{assemblyName}' doesn't exist.");
+                }
+                else
+                {
+                    throw;
+                }
+            }
+
+            return 0;
+        }
+
+        private static int StopDeployment(string[] args)
```

Comment:
So to stop a scale test, would this command have to be called twice, one for each deployment?

Reply:
That's correct, but will change this as discussed below